apps/rundown/src/launcher.ts

```diff
@@ -113,17 +106,13 @@ class Launcher {
     moduleApi.Module.prototype.require = hookedRequire as NodeJS.Require;
     Launcher._copyProperties(hookedRequire, realRequire);
 
+    // Generally 'close' is a better event to handle for child_process cleanup, however in this case
+    // we want to send the IPC response as early as possible.
```

Comment:
`process` does not fire the `close` event; this event handler needs to run before the host process exits.

Reply:
I missed that 😄